src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1069,6 +1081,20 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    // Method Description:
+    // - Close the currently focused tab. Focus will move to the left, if possible.
+    void TerminalPage::_CloseFocusedTabPane()
+    {
+        if (_panes.Size() > 1)
```

Comment:
Unfortunately, `_panes` doesn't exist haha. Good abstraction though.

After a quick look, you should have access to the tab (obviously). From there, you should be able to take a look at the `_rootPane` and just figure out if it's a leaf or not. If it's a leaf, there's only one pane in the tab, otherwise, there's multiple.

Hopefully this helps you get started. 😊

Reply:
`Tab` already has a private `_GetLeafPaneCount` - that could be made public, which would work here

Reply:
So, I have `if (Tab::_rootPane)` but I can't figure out what is supposed to be next. Thanks in advance.

Reply:
Hello?

Reply:
Whoops! Missed this sorry!

So, in Tab.h, move `_GetLeafPaneCount()` to be public (if you haven't done so already).
In TerminalPage, take a look at `_GetActiveControl()`. Here, we can see how you can get a hold of the currently focused tab. Try to replicate that in your new `_CloseFocusedTabPane()` function.
Then, (last step) if you already have the focused tab saved to a variable, call `GetLeafPaneCount()` off of it. Should look something like this:
```c++
if (focusedTab->GetLeafPaneCount() > 1)
```

Reply:
Alright. Thx!

Reply:
`focusedTab` is coming up with error, `identifier "focusedTab" is undefined`.

Reply:
Try this:
```c++
if (auto index{ _GetFocusedTabIndex() })
{
	auto focusedTab{ _GetStrongTabImpl(*index) };
	if (focusedTab->GetLeafPaneCount() > 1)
```

Reply:
`GetFocusedTabIndex` and `GetStrongTabImpl` comes out as error, still undefined

Reply:
Upload the branch and I can take a look?

Reply:
Yes, look at the latest commit

Reply:
Alright. Thanks for the help @carlos-zamora. Resolved.